src/lib/blueprint.ts

```diff
@@ -121,10 +121,293 @@ export interface TypesModule {
   openapi: Openapi
 }
 
-export const createBlueprint = ({ openapi }: TypesModule): Blueprint => {
+export const createBlueprint = (openapi: Openapi): Blueprint => {
+  const isFakeData = openapi.info.title === 'Foo'
+  const targetPath = '/acs/systems/list'
+  const targetSchema = 'acs_system'
+
   return {
     name: openapi.info.title,
-    routes: [],
-    resources: {},
+    routes: createRoutes(openapi.paths, isFakeData, targetPath),
+    resources: createResources(
+      openapi.components.schemas,
+      isFakeData,
+      targetSchema,
+    ),
+  }
+}
+
+const createRoutes = (
+  paths: Openapi['paths'],
+  isFakeData: boolean,
+  targetPath: string,
+): Route[] =>
+  Object.entries(paths)
+    .filter(([path]) => isFakeData || path === targetPath)
+    .map(([path, pathItem]) => createRoute(path, pathItem))
+
+const createRoute = (
+  path: string,
+  pathItem: Record<string, unknown>,
+): Route => ({
+  path,
+  namespace: { path: '/acs' },
+  endpoints: createEndpoints(path, pathItem),
+  subroutes: [],
+})
+
+const createEndpoints = (
+  path: string,
+  pathItem: Record<string, unknown>,
+): Endpoint[] =>
+  Object.entries(pathItem)
+    .filter(
+      ([, operation]) => typeof operation === 'object' && operation !== null,
+    )
+    .map(([method, operation]) =>
+      createEndpoint(method, operation as Record<string, unknown>, path),
+    )
+
+const createEndpoint = (
+  method: string,
+  operation: Record<string, unknown>,
+  path: string,
+): Endpoint => ({
+  name:
+    'operationId' in operation && typeof operation['operationId'] === 'string'
+      ? operation['operationId']
+      : `${method}${path.replace(/\//g, '_')}`,
+  path,
+  methods: [method.toUpperCase() as Method],
+  semanticMethod: method.toUpperCase() as Method,
+  preferredMethod: method.toUpperCase() as Method,
```

Comment:
Prefer

```ts
const method = method.toUpperCase()
```

I too often start with `const foo = () => ({...})` and then have to switch quickly to `const foo = () => { return {...} }` once I have any non-trival one-line logic.